### PR TITLE
HD-1745: Spark build/install script

### PR DESCRIPTION
**Pull request by sookim-altiscale**
This script for jenkins docker job.

### CODE REVIEW

File: scripts/justinstall.sh [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  mkdir -p $dstdir
+  cp $srcname $dstdir
+done
+
+# 2.2 Copy config directory
+ETC_DIR=${INSTALL_DIR}/etc/alti-spark-${ARTIFACT_VERSION}
+mkdir -p --mode 0755 ${ETC_DIR}
+cp -rp ${SPARK_HOME}/conf/* ${ETC_DIR}
+echo "DO NOT HAND EDIT, DEPLOYED BY RPM and CHEF" >  ${ETC_DIR}/DO_NOT_HAND_EDIT.txt
+ln -s /etc/alti-spark-${ARTIFACT_VERSION} ${DISTDIR}/conf
```
Comment (alee-altiscale) [inline scripts/justinstall.sh]: This is managed by chef already, do we still want this here?

File: scripts/justinstall.sh [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+--url ${GITREPO} \
+--license "Apache License v2" \
+-s dir \
+-t rpm \
+-n ${RPM_NAME}  \
+-v ${ALTISCALE_RELEASE} \
+--iteration ${DATE_STRING} \
+--description "${RPM_DESCRIPTION}" \
+${CONFIG_FILES} \
+--rpm-user root \
```
Comment (alee-altiscale) [inline scripts/justinstall.sh:118]: I don't know if this is necessary in Docker, but building as `root` doesn't look trivial to me. Is this how it supposed to be for Docker?

File: scripts/justinstall.sh [hunk 3]
```diff
@@ -0,0 +1,123 @@
+cd $WORKSPACE
+ALTISCALE_RELEASE=${ALTISCALE_RELEASE:-0.1.0}
```
Comment (alee-altiscale) [inline scripts/justinstall.sh]: Shouldn't the default starts with ADC `4.0.0` or something like that? This is referring to Altiscale release right?

Comment (sookim-altiscale) [reply scripts/justinstall.sh]: Yes, However, most of our build script has the default as 0.1.0. I will change it as 4.0.0.

File: scripts/justinstall.sh (hunk 1 above)
Comment (sookim-altiscale) [reply scripts/justinstall.sh]: No, I will remove it.

File: scripts/justinstall.sh (hunk 2 above)
Comment (sookim-altiscale) [reply scripts/justinstall.sh:118]: The rpm-user/rpm-group information use when you install rpm, what is the owner:group of the package files and directory. @alee-altiscale would you tell me which owner:group should own the spark package?
FYI, <owner:group>
1. hadoop package: <hadoop:hadoop>
2. oozie package: <oozie:hadoop>
3. hive package: <root:root>
4. tez package: <root:root>
5. pig package: <root:root>

Comment (alee-altiscale) [reply scripts/justinstall.sh:118]: I see. yeah, LGTM.

**Pull request merged**